changelogs/fragments/20241509-azure_manage_postgresql.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - Adding deadlock_timeout to defualt settings.
```

Comment:
This description is misleading, I think better to describe this with:
"Postgresql Integration test improvement - Add key-value example to postgresql_settings"

Reply:
Hi, Changes applied as suggested